tests/console/kdump_and_crash.pm

```diff
@@ -15,62 +15,11 @@ use strict;
 use warnings;
 use testapi;
 use utils;
-use power_action_utils 'power_action';
 use kdump_utils;
-use version_utils 'is_sle';
-use registration;
-use Utils::Backends 'is_spvm';
 
 sub run {
-    my ($self) = @_;
-    select_console('root-console');
```

Comment:
This is missing
https://openqa.suse.de/tests/3374510#step/kdump_and_crash/1

Reply:
a ticket poo#57074 was filed and a PR will soon be merged.

Reply:
@hjluo Thank you